packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -0,0 +1,286 @@
+
+@Component({
+	selector: `host-component`,
+	template: `
+		<stark-dropdown [options]="options" [label]="label" [placeholder]="placeholder"
+						[value]="value" [selectionChange]="dropdownSelectionChanged"
+						[dropdownId]="dropdownId"></stark-dropdown>`
+})
+class TestHostComponent {
+	@ViewChild(StarkDropdownComponent)
+	public dropdownComponent: StarkDropdownComponent;
+	public dropdownId: string;
+	public header?: string;
+	public label?: string;
+	public options: any[];
+	public placeholder: string;
+	public value: any | any[];
+	public dropdownSelectionChanged: Function;
+	public required?: boolean;
+}
+
+fdescribe("DropdownComponent", () => {
+	let component: StarkDropdownComponent;
+	let hostComponent: TestHostComponent;
+	let hostFixture: ComponentFixture<TestHostComponent>;
+
+	const simpleOptions: string[] = ["1", "2", "3"];
+	const dropdownLabel: string = "dropdown label";
+	const dropdownPlaceholder: string = "dropdown placeholder";
+	const dropdownValue: string = "dropdownValue";
+	const dropdownId: string = "dropdownId";
+	const dropdownOnChange: any = () => {
+		return "dummyDropdownOnChange";
+	};
+
+	beforeEach(async(() => {
+		return TestBed.configureTestingModule({
+			imports: [CommonModule, MatSelectModule, MatOptionModule, TranslateModule.forRoot(),BrowserAnimationsModule],
+			declarations: [StarkDropdownComponent, TestHostComponent],
+			providers: [{ provide: STARK_LOGGING_SERVICE, useValue: new MockStarkLoggingService() }, TranslateService],
+			schemas: [NO_ERRORS_SCHEMA] // tells the Angular compiler to ignore unrecognized elements and attributes (svgIcon)
+		}).compileComponents();
+	}));
+
+	// Inject the mocked services
+	beforeEach(() => {
+		hostFixture = TestBed.createComponent(TestHostComponent);
+		hostComponent = hostFixture.componentInstance;
+		hostFixture.detectChanges();
+		component = hostComponent.dropdownComponent;
+
+		hostComponent.dropdownId = dropdownId;
+		hostComponent.label = dropdownLabel;
+		hostComponent.dropdownSelectionChanged = dropdownOnChange;
+		hostComponent.options = simpleOptions;
+		hostComponent.placeholder = dropdownPlaceholder;
+		hostComponent.value = dropdownValue;
+
+		hostFixture.detectChanges();
+	});
+
+	describe("on initialization", () => {
+		it("should set internal component properties", () => {
+			expect(hostFixture).toBeDefined();
+			expect(component).toBeDefined();
+			expect(component.logger).not.toBeNull();
+			expect(component.logger).toBeDefined();
+		});
+	});
+
+	describe("options defined as array of simple types", () => {
+		it("should render the appropriate content", () => {
+			const dropdownComponent: DebugElement = hostFixture.debugElement.query(By.directive(StarkDropdownComponent));
+			// expect(dropdownComponent.nativeElement.innerHTML).toContain("<label");
+			// const labelElement: any = dropdownComponent.query(By.css("label"));
+			// expect(labelElement.nativeElement.innerHTML).toMatch(dropdownLabel);
```

Comment:
These 3 lines in comments... can they be removed? or if they are needed can you please add a TODO or FIXME ?